parquet-hadoop/src/main/java/org/apache/parquet/crypto/AesCtrDecryptor.java

```diff
@@ -86,6 +87,48 @@ public byte[] decrypt(byte[] ciphertext, int cipherTextOffset, int cipherTextLen
 
     return plainText;
   }
+  public ByteBuffer decrypt(ByteBuffer ciphertext, byte[] AAD) {
```

Comment:
There are many duplicate code from `byte[] decrypt(byte[] ciphertext, int cipherTextOffset, int cipherTextLength, byte[] AAD)`. Is it possible to refactor them to share common logic?